queries/haskell/highlights.scm

```diff
@@ -1,14 +1,23 @@
 ;; ----------------------------------------------------------------------------
-;; Parameters
+;; Parameters and variables
 
 ;; NOTE: These are at the top, so that they have low priority,
 ;; and don't override destructured parameters 
 
+(variable) @variable
+(pat_wildcard) @variable
```

Comment:
What is `pat_wildcard`?

Reply:
Probably `_` in `match x ... _ -> somefunc`

Reply:
:point_up:

Reply:
So `@parameter` maybe?

Reply:
It can appear in any pattern matching expression. I wouldn't change it. I just moved these up from lines 154 and 153, so they don't override the other queries.